components/atoms/AlphaBanner.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+/**
+ * Displays the Alpha AlphaBanner on the page
+ */
+
+export const AlphaBanner = ({ text }) => {
+  return (
+    <div className="bg-gray-100">
+      <div className="lg:container xxs:mx-0 xxs:px-0 lg:mx-auto lg:px-0 xxl:mx-auto">
+        <div className="xxs:w-screen lg:w-2/3 xl:w-1/2 p-4">
```

Comment:
feel like all these divs are unnecessary. Seems like this can be accomplished using two divs or so with the `layout-container` css class

Reply:
With the new commit, I managed to remove one div. Not sure how to remove more :s